applications/new_schema.py

```diff
@@ -71,6 +76,34 @@ def get_node(cls, info, id):
         return super().get_node(info, id)
 
 
+class UpdateBerthApplication(graphene.ClientIDMutation):
+    class Input:
+        berth_application_id = graphene.ID(required=True)
```

Comment:
in the resources app you used just `id` field for all the update mutations. I suggest you do the same here and we adopt it as a convention 🙂

Reply:
True, it was a leftover from my original idea of having a `AssignApplicationToCustomer` mutation. Good catch.